code/modules/projectiles/guns/projectile/constructable/siegecannon.dm

```diff
@@ -176,7 +176,7 @@
 	if(do_after(user, src, 3 SECONDS))
 		beenClowned = TRUE
 		icon_state = "clownnon"
-		name = "Circus Cannon"
+		name = "circus Cannon"
```

Comment:
cannon is also not a proper noun

Reply:
Cannon is its genus

Reply:
Does that mean the circus Cannon is alive? Does it honk upon firing?

Reply:
Now it does

Reply:
> 
> 
> Now it does

Impossibly based.